scripts/asm-installer/install_asm

```diff
@@ -1715,6 +1755,20 @@ install_secrets() {
     --from-file="${CA_CHAIN}"
 }
 
+init_private_ca() {
+  WORKLOAD_IDENTITY="$PROJECT_ID.svc.id.goog[istio-system/istiod-service-account]"
+
+  local NAME; NAME=$(echo "${CA_NAME}" | cut -f6 -d/)
+  local CA_LOCATION; CA_LOCATION=$(echo "${CA_NAME}" | cut -f4 -d/)
+  local CA_PROJECT; CA_PROJECT=$(echo "${CA_NAME}" | cut -f2 -d/)
+
+  retry 3 run gcloud beta privateca subordinates add-iam-policy-binding "${NAME}" \
```

Comment:
I'm not sure what this is doing, but it looks like this should only be required per project and not per cluster, right? Can we make the function name a bit more descriptive? Is there a way to check if this is already done before attempting to add the policy binding? Otherwise the caller requires permissions to modify Private CA even if it's already set up--this is the reason for all of the individual `--enable...` flags.

Reply:
I agree that the caller needs permission to modify Private CA IAM. Will add that as well. 

However, this script (as of now) is being called per cluster. In our docs, we mention that we want a sub-ordinate CA per cluster. Therefore, the expectation is that the IAM policy binding command will need to be called each time this script is invoked, even if from the same project

Reply:
How does this command know what cluster it's attached to?

Reply:
It doesn't, but the CA_NAME is expected to be unique per cluster

Reply:
Sounds good, thanks!